packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.scss

```diff
@@ -155,7 +159,7 @@
 }
 
 //Primary button style override
-.root .primaryButton {
+.primaryButton {
```

Comment:
removing specificity here could cause issues.

Reply:
agreed...i'm guessing these styles are overriding previous styles...probably applied via a classname, so any change in load order could break this.

Reply:
If you want to simplify this, you could pass the styles into the button styles prop